document/core/appendix/changes.rst

```diff
@@ -104,17 +104,52 @@ Added instructions that modify ranges of memory or table entries [#proposal-reft
 * Active data and element segments boundaries are no longer checked at compile time but may trap instead
 
 
+.. index:: instructions, SIMD, value type, vector type
+
+Vector instructions
+...................
+
+Added vector type and instructions that manipulate multiple numeric values in parallel (also known as _SIMD_, single instruction multiple data) [#proposal-vectype]_
```

Comment:
what is `_SIMD_` supposed to type set as? I see it on the HTML as `_SIMD_`, is it supposed to underline/italics?

Reply:
Fixed.